### PR TITLE
Add tubCloud Latest

### PR DESCRIPTION
**Pull request by tilowestermann**

### CODE REVIEW

**Comment by adityadalal924** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git)

File: Casks/tubcloud.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  url 'https://updates.owncloud.tu-berlin.de/tubcloud-latest-macos.pkg'
+  name 'tubCloud Client'
+  homepage 'https://www.tubit.tu-berlin.de/menue/dienste/daten_server/tubcloud/tubcloud_sync-client/'
+  license :gpl
+
+  pkg 'tubcloud-latest-macos.pkg'
+
+  uninstall :pkgutil => 'de.tu-berlin.tubit.owncloud'
+
```
Comment (vitorgalvao) [inline Casks/tubcloud.rb]: Please remove this empty line.

**Comment by tilowestermann** [comment]
Done. Thanks for the suggestion.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as e639d62737b726d9e162238a78e0071b571ec3d5 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**